api/src/pcapi/routes/pro/statistics.py

```diff
@@ -0,0 +1,30 @@
+from flask_login import current_user
+from flask_login import login_required
+
+from pcapi.connectors.clickhouse import queries as clickhouse_queries
+from pcapi.models.api_errors import ApiErrors
+from pcapi.routes.apis import private_api
+from pcapi.routes.serialization.statistics_serialize import StatisticsModel
+from pcapi.routes.serialization.statistics_serialize import StatisticsQueryModel
+from pcapi.serialization.decorator import spectree_serialize
+from pcapi.utils.rest import check_user_has_access_to_venues
+
+from . import blueprint
+
+
+@private_api.route("/get_statistics", methods=["GET"])
```

Comment:
Les routes de `private_api` séparent les mots par des `-` et non des`_`, donc par homogénéité : 
```suggestion
@private_api.route("/get-statistics", methods=["GET"])
```